### ISSUE TITLE
Subsite Pages Header Issue Above 1920 width.

### ISSUE THREAD

**Issue by montelco** [montelco/rcgc-business-studies#1]
Image was not responding. To resolve this, the changes were made having the inset area as a background element. This, however, causes an issue with total accessibility. Additionally, positioning content within this inset area causes problems relating to how the information should be appropriately scaled and cropped.

**Comment by montelco** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Fix scaling on Subpage Header](https://trello.com/c/ob2QmRni/47-fix-scaling-on-subpage-header)
